global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
apscheduler.executors.default - run_job(line:131) - Job "training_phase (trigger: interval[0:00:01], next run at: 2024-04-04 05:09:10 UTC)" raised an exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/app/global_controller.py", line 1347, in training_phase
    if os.path.getsize(trainig_input_trace_file) == 0:
  File "/usr/local/lib/python3.10/genericpath.py", line 50, in getsize
    return os.stat(filename).st_size
FileNotFoundError: [Errno 2] No such file or directory: 'trace.csv'
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
__main__ - filter_incomplete_traces(line:1292) - !!! ERROR: len(complete_traces[us-east-1]) == 0
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(505) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 515
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(509) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 519
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(472) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 482
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(532) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 542
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(509) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 519
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(499) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 509
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(459) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 469
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(491) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 501
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(527) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 537
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(486) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 496
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(532) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 542
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(461) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 471
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(470) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 480
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(477) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 487
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(502) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 512
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(523) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 533
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(521) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 531
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(483) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 493
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(480) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 490
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(507) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 517
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(485) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 495
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(464) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 474
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(496) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 506
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(494) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 504
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(508) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 518
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(521) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 531
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(472) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 482
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(473) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 483
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(509) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 519
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(510) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 520
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(508) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 518
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(514) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 524
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(499) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 509
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(516) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 526
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(511) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 521
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(476) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 486
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(485) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 495
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(457) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 467
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(504) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 514
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(501) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 511
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(506) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 516
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(495) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 505
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(474) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 484
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(523) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 533
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(492) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 502
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(493) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 503
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(455) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 465
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(455) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 465
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(908) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 918
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(536) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 546
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(539) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 549
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(545) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 555
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(477) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 487
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(517) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 527
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(515) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 525
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(495) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 505
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(519) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 529
